brave/browser-laptop
# Use pkg installer for macOS in addition to of dmg image

On the 'dev' channel only (which equates to the Release channel), output a pkg file as well as dmg. All other channels remain as dmg for now.

Fix #12784

**Update:** I've included a commit which upgrades ` electron-builder` from 17.1.1 to 19.55.3. This has an enhanced support for pkg and solves some issues I saw with pkgs in 17.1.1:
 - sometimes the install process would not result in /Applications/Brave.app, if it detected another Brave app (named anything) _anywhere_ on your computer - instead it would upgrade that found .app.
 - there was an extra step for 'choose your disk' in 17.1.1. With 19.55.3 that step is skipped and instead the user can change location from the default disk if they really want, but the installer won't encourage it.
Left as a separate commit in case the risk of upgrading is not desired.


Submitter Checklist:




### Test plan
- Delete all Brave.app's everywhere (/Applications, browser-laptop/Brave*/, ...)
  - the .pkg installer will see these as existing (and moved installations), and upgrade them instead)
- Build package
- Build installer
- Run installer

Observe:
- Brave still runs correctly after install


Tests
